specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/schema/http.schema.patch.json

```diff
@@ -434,7 +443,7 @@
                 {
                     "^[a-zA-Z]+[a-zA-Z0-9\\._\\-]*$":
                     {
-                        "title": "Authorization",
+                        "title": "Guard",
```

Comment:
Not sure about this one. I think the structure of what follows might be a definition of an authorization, not a guard.

Reply:
IMHO, a guard is a guard, this is a reference to a guard used to provide context for an authorization. 😄

Reply:
Well whichever one you want I will put in both the MQTT schema and HTTP schema